When building on ubuntu-18 missing pre-preprocessor symbol HUGE

```
HDFEOS2.cc: In member function ‘void HDFEOS2::File::handle_grid_cf_attrs()’:
HDFEOS2.cc:1769:43: error: ‘HUGE’ was not declared in this scope
 1769 |                     float tempfillvalue = HUGE;
      |                                           ^~~~

```

From a phone conversation (Jim and I) we believe this is related to an obsoleted value.h pre-processor symbol.

We believe it should be changed to MAXFLOAT.

See https://github.com/OPENDAP/hyrax/issues/10

--- a/HDFEOS2.cc
+++ b/HDFEOS2.cc
@@ -1766,7 +1766,7 @@ void File::handle_grid_cf_attrs() throw(Exception) {
             // We may find such cases for other products and will tackle them also.
             if (true == (*i)->addfvalueattr) {
                 if((((*j)->getFillValue()).empty()) && ((*j)->getType()==DFNT_FLOAT32 )) {
-                    float tempfillvalue = HUGE;
+                    float tempfillvalue = MAXFLOAT;
                     (*j)->addFillValue(tempfillvalue);
                     (*j)->setAddedFillValue(true);
                 }

